basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/model/City.java

```diff
@@ -0,0 +1,44 @@
+package fr.openwide.core.basicapp.core.business.referential.model;
+
+import javax.persistence.Cacheable;
+import javax.persistence.Entity;
+
+import org.bindgen.Bindable;
+import org.hibernate.search.annotations.Analyzer;
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Indexed;
+
+import fr.openwide.core.jpa.more.business.generic.model.GenericListItem;
+import fr.openwide.core.jpa.search.util.HibernateSearchAnalyzer;
+import fr.openwide.core.spring.util.StringUtils;
+
+@Entity
+@Bindable
+@Indexed
+@Cacheable
+public class City extends GenericListItem<City> {
+
+	private static final long serialVersionUID = -5714475132350205234L;
+
+	public static final String LABEL_AUTOCOMPLETE = "labelAutocomplete";
+	
+	public City() {
+		setPosition(0);
```

Comment:
Setting the position is not required anymore when you don't use it. Default is 0. Same below.